redwood-layout-api/src/commonMain/kotlin/app/cash/redwood/layout/api/properties.kt

```diff
@@ -146,3 +146,17 @@ public fun Margin(
 public fun Margin(
   all: Int = 0,
 ): Margin = Margin(all, all, all, all)
+
+// Temporary constructor before migrating everything to doubles.
+@Stable
+public fun Margin(
+  left: Double = 0.0,
```

Comment:
Meta comment: no start/end?

Reply:
Yep! Will fix this soon. I changed this to `left`/`right` as I'm pretty sure we're not handling LTR properly in some spots.

Reply:
For context, there was a bit of a discussion about it [here](https://github.com/cashapp/redwood/pull/867#issuecomment-1482573336)

Reply:
Ah cool, thanks!